packages/fluentui/e2e/cypress/support/commands.js

```diff
@@ -44,7 +44,7 @@ Cypress.Commands.add('focusOn', selector => {
   cy.get(selector).focus();
 });
 
-Cypress.Commands.add('hover', selector => {
+Cypress.Commands.add('hoverOn', selector => {
```

Comment:
The newest version of cypress now has its own `cy.hover` command so I had to rename this one to avoid conflict